src/api/app/policies/comment_policy.rb

```diff
@@ -31,4 +31,11 @@ def update?
   def reply?
     !(user.blank? || user.is_nobody? || record.user.is_nobody?)
   end
+
+  # Only logged-in Admins can moderate non-deleted comments
```

Comment:
This makes little sense I guess. No can should be able to moderate soft-deleted comments...